Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -454,7 +454,8 @@ struct JPEGLoadingContext {
 
     State state { State::NotDecoded };
 
-    Array<Optional<Array<u16, 64>>, 4> quantization_tables {};
+    Array<Array<u16, 64>, 4> quantization_tables {};
+    Vector<u8, 4> registered_quantization_tables {};
```

Comment:
If you make this an `Array<bool, 4>` where index i stores if the i'th table is present, I think the code will become a bit shorter and simpler.

Reply:
Indeed, this is one line shorter!